package CfgTitle;
import "CfgCommon.proto";

message Title {
	required uint32 id = 1;	// 称号id
	required uint32 itemId = 2; // 激活道具id
	required string name = 3;	// 称号名字
	required uint32 limitTime = 4; // 称号时效（小时）
	repeated CfgCommon.Attribute attr = 5; // 称号增加属性
}

// 称号数组 根据定义好的json文件配置
message Titles {
	repeated Title title = 1;
} 